extensions/assessmentExtension/context.json

```diff
@@ -4,12 +4,20 @@
     "extensions": "https://w3id.org/openbadges/extensions#Assessment",
     "schema": "http://schema.org/",
     "overallDescription": "extensions:overallDescription",
-    "assessment": "extensions:assessment"
```

Comment:
"assessment" shouldn't be removed from the context. All shorthand (IRI) terms used in the examples and users' implementations must appear in a context to contribute to the linked data value of the document.